public/usage-examples/graphics/fill_triangle_on_window/fill_triangle_on_window-1-simple-oop.cs

```diff
@@ -0,0 +1,19 @@
+using SplashKitSDK;
+
+namespace Program
+{
+  public class Program
+  {
+    public static void Main()
+    {
+      Window myWindow = SplashKit.OpenWindow("Fill Triangle On Window", 800, 600);
+
+      // Fill triangle on the window and refresh
+      SplashKit.FillTriangleOnWindow(myWindow, SplashKit.ColorBlue(), 300, 250, 500, 250, 400, 350);
+      SplashKit.RefreshWindow(myWindow);
+
+      SplashKit.Delay(3000);
```

Comment:
Please review the guide on how to convert from top level to OOP  in line 9 and 16

https://thoth-tech.netlify.app/products/splashkit/splashkit-website/tutorials-documentation/04-oop-styling/